MTV-3148: Release Notes and Attributes for MTV 2.9.3 (#742)
kubev2v/forklift-documentation: documentation/modules/rn-2-9-3-resolved-issues.adoc

// Module included in the following assemblies:
//
// * documentation/doc-Release_notes/master.adoc

:_content-type: CONCEPT
[id="resolved-issues-2-9-3_{context}"]
= Resolved issues 2.9.3

.NetworkMap referencing incorrect namespace for NAD in migration plan

In previous versions of {project-short} 2.9, the NetworkMap in {project-short} 2.9.1 incorrectly referenced a namespace for Network Address Determination (Network Address Determination System, or NADS) in migration plans. As a consequence, migration plans failed to complete. With the release of {project-short} 2.9.3, the NetworkMap now correctly references the NADS namespace, resolving migration plan failures. As a result, migration plans successfully map VM networks to NADS, enhancing the user's experience.
// red hat employee only so not adding link

.Timeouts while querying inventory due to insufficient route timeout setting

> In previous versions of {project-short} 2.9, insufficient route timeout setting during inventory querying led to timeouts, which in turn affected user experience by blocking migration progress. With the release of {project-short} 2.9.3, for {project-short} deployment, the inventory route timeout has been actively set to 360 seconds to resolve VM migration timeouts, enabling seamless migration of VMs. link:https://issues.redhat.com/browse/MTV-3107[(MTV-3107)]

Author: ```suggestion
In previous versions of {project-short} 2.9, insufficient route timeout settings during inventory querying led to timeouts, which in turn affected user experience by blocking migration progress. Therefore, users could not choose a VM to migrate because the inventory server timed out before they could choose a VM from the inventory, and they were unable to create a migration plan. With the release of {project-short} 2.9.3, for {project-short} deployment, the inventory route timeout has been actively set to 360 seconds to resolve VM migration timeouts, enabling seamless migration of VMs. link:https://issues.redhat.com/browse/MTV-3107[(MTV-3107)]
```

// Module included in the following assemblies:
//
// * documentation/doc-Release_notes/master.adoc

:_content-type: CONCEPT
[id="resolved-issues-2-9-3_{context}"]
= Resolved issues 2.9.3

.Users were unable to use the VMware file name when creating a PVC name template

> In previous versions of {project-first} 2.9, users encountered difficulties when using VMware file names to create PVC name templates, necessitating manual workarounds for PVC names with VMware vmdk file names. With the release of {project-short} 2.9.3, support for VMware vmdk file names in PVC name templates has been implemented. As a result, end users now have the ability to customize PVC names with VMware vmdk file names. link:https://issues.redhat.com/browse/MTV-3091[(MTV-3091)]

Reviewer: > necessitating manual workarounds for PVC names with VMware vmdk file names

we didn't find a workaround, so I think we can omit this

Author: @yaacov it is on Stefan's list so i am going to leave it, if that is ok - https://redhat-internal.slack.com/archives/C07HDAK66QG/p1755766244909769?thread_ts=1755629359.452669&cid=C07HDAK66QG

.NetworkMap referencing incorrect namespace for NAD in migration plan

In previous versions of {project-short} 2.9, the NetworkMap in {project-short} 2.9.1 incorrectly referenced a namespace for Network Address Determination (Network Address Determination System, or NADS) in migration plans. As a consequence, migration plans failed to complete. With the release of {project-short} 2.9.3, the NetworkMap now correctly references the NADS namespace, resolving migration plan failures. As a result, migration plans successfully map VM networks to NADS, enhancing the user's experience.
// red hat employee only so not adding link

.Timeouts while querying inventory due to insufficient route timeout setting

> In previous versions of {project-short} 2.9, insufficient route timeout setting during inventory querying led to timeouts, which in turn affected user experience by blocking migration progress. With the release of {project-short} 2.9.3, for {project-short} deployment, the inventory route timeout has been actively set to 360 seconds to resolve VM migration timeouts, enabling seamless migration of VMs. link:https://issues.redhat.com/browse/MTV-3107[(MTV-3107)]

Reviewer: once they did choose a vm and created a plan, the migration progress was fine

.Repeated failed calls to incorrect endpoints

In previous versions of {project-short}, incorrect endpoint path formatting in the `useNetworks` hook led to repeated failed calls. This resulted in repeated failures when creating migration plans due to incorrect endpoint calls, particularly in the 'Virtual machines' section of the Migration plan creation wizard. With this release, we've fixed the repeated incorrect endpoint calls in the 'Virtual machines' section, improving the creation of migration plans by eliminating failed calls and ensuring a smoother user experience. link:https://issues.redhat.com/browse/MTV-2978[(MTV-2978)]

.Users unable to match VMware file names with PVC names, causing inconsistency in resource identification

> In previous versions of {project-short}, users encountered an issue where they could not use VMware file names in Persistent Volume Claim (PVC) name templates, leading to inconsistent resource identification due to mismatched names. This problem stemmed from the inability to utilize VMware vmdk file names in PVC name templates. With this release, we have implemented the functionality to allow users to incorporate VMware VMDK file names into PVC name templates. As a result, users now have the ability to customize PVC names with VMware VMDK file names, thereby ensuring naming consistency. link:https://issues.redhat.com/browse/MTV-3091[(MTV-3091)]

Author: ```suggestion
In previous versions of {project-short}, users encountered an issue where they could not use VMware file names in Persistent Volume Claim (PVC) name templates, leading to inconsistent resource identification due to mismatched names. This problem stemmed from the inability to use VMware VMDK file names in PVC name templates. With this release, we have implemented the functionality to allow users to incorporate VMware VMDK file names into PVC name templates. As a result, users now have the ability to customize PVC names with VMware VMDK file names, thereby ensuring naming consistency. link:https://issues.redhat.com/browse/MTV-3091[(MTV-3091)]
```

In previous versions of {project-short} 2.9, {virt} encountered MAC conflicts during migration due to OVA files downloaded from VMware not having MAC addresses. This resulted in OVA migration failures due to empty MAC addresses. With the release of {project-short} 2.9.3, we have implemented a solution that disregards empty MAC addresses during OVA migration checks, thereby eliminating MAC conflict errors caused by empty MAC addresses, and thus improving OVA migration success rates. link:https://issues.redhat.com/browse/MTV-2476[(MTV-2476)]

.Incorrect v2v command on {project-short} 2.8.0 with RHEL8.9, causing guest migration failure

In previous versions of {project-short}, a wrong v2v command on {project-short} 2.8.0 with RHEL8.9 on a secure ESXi provider led to guest migration failure, preventing users from migrating guests from secure ESXi. Consequently, failed migrations occurred. With the release of {project-short} 2.9.3, the issue with the ESXi migration command on MTV has been addressed, allowing MTV to correctly migrate guests from secure ESXi. This improvement enhances the efficiency of virtual machine migration.

> .{project-short} overwrites firstboot.bat with its own copy

Author: ```suggestion

.{project-short} overwrites firstboot.bat with its own copy
```

.Incorrect v2v command on {project-short} 2.8.0 with RHEL8.9, causing guest migration failure

In previous versions of {project-short}, a wrong v2v command on {project-short} 2.8.0 with RHEL8.9 on a secure ESXi provider led to guest migration failure, preventing users from migrating guests from secure ESXi. Consequently, failed migrations occurred. With the release of {project-short} 2.9.3, the issue with the ESXi migration command on MTV has been addressed, allowing MTV to correctly migrate guests from secure ESXi. This improvement enhances the efficiency of virtual machine migration.

.{project-short} overwrites firstboot.bat with its own copy

> In previous versions of {project-short}, the script `firstboot.bat` in the Windows directory was not being actively deleted, leading to repeated execution of firstboot scripts on later boots. Consequently, user experience was degraded. With this release, the redundant firstboot script has been actively removed from the Windows directory. link:https://issues.redhat.com/browse/MTV-3093[(MTV-3093)]

Author: @solenoci & @yaacov - we good to merge?

Reviewer: from the issue:
"The firstboot script already exists, and it seems that MTV is overwriting the new one with an older version."

so the issue was:
MTV was overwriting the firstboot script

fix is:
MTV stopped overwriting the firstboot script

> In previous versions of {project-short}, the script `firstboot.bat` in the Windows directory was not being actively deleted, leading to repeated execution of firstboot scripts on later boots. Consequently, user experience was degraded. With this release, the redundant firstboot script has been actively removed from the Windows directory. link:https://issues.redhat.com/browse/MTV-3093[(MTV-3093)]

Author: ```suggestion
In previous versions of {project-short}, {project-short} overwrote the `firstboot.bat` script in Windows. Consequently, user experience was degraded. With this release, the redundant firstboot script is no longer overwritten. link:https://issues.redhat.com/browse/MTV-3093[(MTV-3093)]
```
